### DISCUSSION TITLE
Support for Dot Net / .Net Language

### DISCUSSION THREAD

**Discussion by Serena1227** [ls1intum/Artemis#4727] [Q&A]
Hello, I have a question: Can I create more Dot Net / .Net language? If yes, how to create it. Please support me, thank you!

**Comment by krusche** [comment]
Yea, this is possible. I suggest you create a Java project and you then adapt the 3 repositories (template, test, solution) to use the corresponding C# code.
You also need to adapt the 2 build plans (BASE, SOLUTION) and you need to define a docker image with the environment needed to build and test C# code.

We are interested to support this as a Template in artemis so that instructors can create this more easily in the future. When you have a working solution, please share it with us
